README.md

```diff
@@ -217,6 +252,46 @@ Each cluster that will be connected must have Submariner installed within it. Yo
    |\<SERVICE_CIDR>|Service CIDR for Cluster|""|`10.43.0.0/16`|
    |\<NAT_ENABLED>|If in a cloud provider that uses 1:1 NAT between instances (for example, AWS VPC), you should set this to `true` so that Submariner is aware of the 1:1 NAT condition.|"false"|`false`|
 
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md).
```

Comment:
This paragraph is not necessary IMO.

Reply:
I mean "Current implementation builds submariner operator and deploys submariner in a [Kind setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md)."

Reply:
Someone new might get confused. Its just about being more descriptive.

Reply:
@Jaanki but generally the main README of a project does not have to overly descriptive, that will only confuse people. And also, all that will change a lot soon...

Reply:
Done